src/esm_runscripts/config_initialization.py

```diff
@@ -165,6 +165,12 @@ def get_user_config_from_command_line(command_line_config):
             f"``{command_line_config['runscript_abspath']}`` from the command line.",
         )
 
+    # If the use_env variable defined by the command line is None deleted so that
+    # the definition from the runscript prevails after the user_config.update
+    # If it's not None (True or False) the definition of the command line wins
+    # over the runscript as is the case for all the other variables
+    if command_line_config["use_venv"] is None:
+        del command_line_config["use_venv"]
```

Comment:
I'm not sure I fully understand this. Does the `use_venv` check for the *presence* of the flag? Because if it only checks if the value is somehow something that evaluates to `True`, this is not really needed. `None` is a `False` thing (well.....technically, not really, they are showing different programming things, but an `if` check will show you the same answer)


```python
>>> my_false_var = False
>>> my_true_var = True
>>> my_none_var = None
>>> if my_false_var:
...     print("woohoo")
...
>>> if my_true_var:
...     print("woohoo2")
...
woohoo2
>>> if my_none_var:
...     print("woohoo3")
...
```